app/src/main/java/fr/free/nrw/commons/upload/DescriptionsAdapter.java

```diff
@@ -122,7 +125,20 @@ public void init(int position) {
                 });
 
             } else {
-                descItemEditText.setCompoundDrawablesWithIntrinsicBounds(null, null, null, null);
+                descItemEditText.setCompoundDrawablesWithIntrinsicBounds(null, null,
+                    getIcon(R.drawable.ic_clear_24dp), null);
+                descItemEditText.setCompoundDrawablePadding(10);
```

Comment:
surely we should be able to make this dp `resources.getDimensionPxelSize(R.dimen.whatever_dimen_is_10dp)`